CommunityFeatures.md

```diff
@@ -0,0 +1,40 @@
+# Introduction
+
+Every time a Pull request improves the community firmware it shall note down it's achievements and usage in this document.
+
+
+# General improvements
+
+Here is a list of general improvements that have been made as a list ordere by date:
+
+* 2023-06-17 - This list was started and it's first entry shall be replaced with the first improvement
+
+
+# Added features
+
+Here is a list of features that have been added to the firmware as a list ordere by date:

```

Comment:
CHANGELOG.md might be a good place for these. It should probably be kept to actual code changes rather than workflow, DBT, etc. Maybe something like CHANGELOG-TOOLS.md or something for the periphery.

Reply:
The idea of CommunityFeatures.md was more as a manual. Do you suggest renaming it to CHANGELOG.md or adding a CHANGELOG(-TOOLS).md in addition? Or rather splitting the file in CHANGELOG.md for the first two headlines and CommunityFeatures.md for the other two?

Reply:
DBT is starting to become a beast and while I've tried to make it self-documenting, I'll probably roll a DBT.md soon or replace some of the instructions in README to better document it.

Consider my comment withdrawn. =)